server/jetstream_api.go

```diff
@@ -1734,7 +1734,7 @@ func (c *client) auditUser() string {
 
 // Returns the audit client name, which will just be IP:Port
 func (c *client) auditClient() (string, int) {
-	parts := strings.Split(c.ncs, " ")
+	parts := strings.Split(c.String(), " ")
```

Comment:
This shows that changes to c.ncs could be risky if some part of code assumes that ncs is simply ip:port..

Reply:
This was the only use of `ncs` directly so that is why i changed it to not reference ncs directly since that seems to be used purely for the `String()` method. It would seem inadvisable to me to try to operate on the result of the String method for virtually any object as the use of the method implies it is the string representation of the object and that could vary depending on the properties of the object (note: I'm not saying this wasn't done, just that it seems inadvisable to me).

I also tried to find all the uses of the String method for the connection but that was difficult and I may have missed some but I believe this is the only one using it operationally rather than simply for logging or similar normal string functions.